src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { PropsWithChildren } from "react";
+
+type IntrinsicElements = JSX.IntrinsicElements["h4"];
+
+interface Props extends IntrinsicElements {
+  disabled?: boolean;
+  id: string;
+  tab: string;
```

Comment:
Very small suggestion, maybe we can give this prop a little more clarity for this variable? tabTitle/tabText? Just a thought though, I saw CMS design was doing the same though